CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
   * `-flusher.wal-dir` for the WAL directory to recover from.
   * `-flusher.concurrent-flushes` for number of concurrent flushes.
   * `-flusher.flush-op-timeout` is duration after which a flush should timeout.
+* [ENHANCEMENT] Move files from /storage/tsdb to a more generic folder i.e /storage/backend. #2262
```

Comment:
Changelog is for things that may impact an end-user; it's better to leave out internal refactoring to make the document smaller hence easier to read.

Reply:
@bboreham Will update this to something that makes better sense.

Reply:
@jaybatra26 No need to add anything to the changelog.

I also agree with @bboreham, we should move those packages as well. Up to you Jay if you want to do that in this PR.

@bboreham #2262 lays out the reasoning behind this. This PR is the first step in a series of PRs to reuse an external object client rolling our own.

Reply:
Sorry, I'm not used to looking in the title of a PR to see what issue(s) it relates to; I expect that in the description.

Reply:
@bboreham, I have updated the description.